models/api.models.js

```diff
@@ -1,3 +1,9 @@
 const db = require("../db/connection");
+const { readFile } = require("fs/promises")
 
-exports.fetchEndpoints = () => {};
+exports.fetchEndpoints = () => {
+    return readFile(`${__dirname}/../endpoints.json`)
+    .then((response) => {
+        return JSON.parse(response)
+    })
+};
```

Comment:
We should actually not need the model at all for this task - all of the logic can be handled in the controller